ruby/tests/generated_code.proto

```diff
@@ -53,6 +53,13 @@ message TestMessage {
   map<string, bool> map_string_bool = 70;
 
   message NestedMessage {
+    message Hello {
```

Comment:
Does this cause problem without your fix? I thought a lower letter started name causes the problem.